packages/core/src/utils.ts

```diff
@@ -242,6 +242,9 @@ export function arrayBufferToBase64(arrayBuffer: ArrayBuffer): string {
 }
 
 export function capitalize(word: string): string {
+  if (word.toLowerCase() === 'id') {
+    return 'ID';
+  }
   return word.charAt(0).toUpperCase() + word.substr(1);
```

Comment:
Is this something we need to be writing from scratch? Are there not any built-in node string packages that handle this?

Reply:
Also, really what we're doing here is "title casing" the string. `Captilalize` suggests capitalizing every character

Reply:
Not many good library options.  This looks like one of the better ones, and it isn't particularly popular.

https://www.npmjs.com/package/capitalize

If the code smell is concerning, we could allow the default behavior and display "id" as "Id"

Reply:
There are a couple of libraries with a bunch of helpful string functions. 

- [String.js ](https://www.npmjs.com/package/) Pretty minimal, no dependencies, and has the `humanize` and `titleCase` that can be used to simplify the `buildFieldNameString` function
- [Humanize Plus](https://github.com/HubSpot/humanize) has a bunch of nifty string normalization functions, and it's `titleCase` function is a bit more advanced in that it doesn't capitalize words like `the`
-